packages/core/src/physics/joint/FixedJoint.ts

```diff
@@ -0,0 +1,43 @@
+import { Joint } from "./Joint";
+import { IFixedJoint } from "@oasis-engine/design";
+import { PhysicsManager } from "../PhysicsManager";
+import { dependentComponents } from "../../ComponentsDependencies";
+import { Collider } from "../Collider";
+import { Vector3 } from "@oasis-engine/math";
+
+/*
+ * A fixed joint permits no relative movement between two bodies. ie the bodies are glued together.
```

Comment:
comments should: `bodies` -> `colliders`

Reply:
done